CHANGELOG.md

```diff
@@ -3,4 +3,4 @@
 ## 1.0.0 (IN PROGRESS)
 * UI app created. Refs UIDEXP-8.
 * Handle security issue because of eslint version, remove excess eslint rules, upgrade babel and eslint-config-stripes versions. Refs UIDEXP-17.
-* Add dependency on @folio/stripes-data-transfer-components. Refs UIDEXP-11.
+* Add dependency on @folio/stripes-data-transfer-components. Add file uploader feature on landing page. Refs UIDEXP-11.
```

Comment:
As I mentioned before, there are too many different changes under UIDEXP-11 (For example, project setup and etc). I think, the current approach should be rethought

Reply:
Agree, we have already agreed to split stories for further cases like this.